wgpu/utils/imgui/imgui_renderer.py

```diff
@@ -152,11 +154,17 @@ def _on_resize(self, event):
     def _on_mouse_move(self, event):
         self._backend.io.add_mouse_pos_event(event["x"], event["y"])
 
+        if self._backend.io.want_capture_mouse:
```

Comment:
Is `want_capture_mouse`  a think from imgui?

Reply:
Yes, See: https://github.com/ocornut/imgui/blob/master/docs/FAQ.md#q-how-can-i-tell-whether-to-dispatch-mousekeyboard-to-dear-imgui-or-my-application.